hexrd/ui/utils/__init__.py

```diff
@@ -482,12 +482,20 @@ def convert_panel_buffer_to_2d_array(panel):
         # Just make a panel buffer with all True values
         panel.panel_buffer = np.ones(panel.shape, dtype=np.bool)
     elif panel.panel_buffer.shape == (2,):
-        # The two integers are specifying the borders in x and y
-        borders = panel.panel_buffer.astype(int)
+        # The two floats are specifying the borders in mm for x and y.
+        # Convert to pixel borders. Swap x and y so we have i, j in pixels.
+        borders = np.round([
+            panel.panel_buffer[1] / panel.pixel_size_row,
+            panel.panel_buffer[0] / panel.pixel_size_col,
+        ]).astype(int)
```

Comment:
I need help understanding the logic of switching the order, but I trust your judgment.

Reply:
So, for instance, in Cartesian coordinates, the `x` is from left to right. But if we convert that to pixels, the columns (`j`) is what is from left to right instead.

So when we go from `xy` => `ij`, we need to swap `x` and `y`. But I'm happy to verify this too.

Reply:
Yes I just verified that this works as expected. For an example, I set the `x` panel buffer to roughly about 44% of the width of the panel. You can see that after using our conversion function to a 2D panel buffer, a large chunk from both the left and right side is `False`:

![example](https://user-images.githubusercontent.com/9558430/215300479-3a4359a2-8bf6-4da6-8124-848c651caebf.png)